src/GovUk.Education.ExploreEducationStatistics.Data.Model/Services/FootnoteService.cs

```diff
@@ -122,25 +133,34 @@ public Footnote GetFootnote(long id)
                 indicatorListParam,
                 filterItemListParam);
         }
-
-        private Subject CreateSubjectLink(Footnote footnote, long subjectId)
+        
+        public IEnumerable<Footnote> GetFootnotes(Guid releaseId)
         {
-            var subject = _subjectService.Find(subjectId, new List<Expression<Func<Subject, object>>>
-                          {
-                              s => s.Footnotes
-                          }) ??
-                          throw new ArgumentException("Subject not found", nameof(subjectId));
-
-            var links = subject.Footnotes;
-            EnsureInitialised(ref links);
+            return _context.SubjectFootnote
+                .Where(footnote => footnote.Subject.ReleaseId == releaseId)
+                .Select(footnote => footnote.Footnote)
+                .Distinct()
+                .Include(footnote => footnote.Filters)
```

Comment:
Post-235 merge, I don't think that these "Include()" need to come before the "Where()" clause, as they're including things that are children of the items in the main Where clause.  Worth re-testing post-235 though!

Reply:
Thanks, I will test this again after the upgrade!